packages/link-bins/src/index.ts

```diff
@@ -120,6 +121,14 @@ async function getPackageBins (
     return []
   }
 
+  if (R.isEmpty(pkg.bin) || !pkg.bin) {
+    throw new PnpmError('INVALID_PACKAGE_BIN', `Package in ${target} must have a non-empty bin field to get bin linked.`)
```

Comment:
If you normalize target here, then normalizePath is not needed in the tests. Otherwise, it doesn't matter if you normalize the expected result. It will fail. The actual result will have `\` on Windows, if the target is not normalized here

Reply:
ah!

Reply:
wait I think target is always normalized before it pass to `getPackageBins`:
```javascript
(await Promise.all(
      pkgDirs
        .map((dir) => path.resolve(modules, dir))
        .filter((dir) => !isSubdir(dir, binPath)) // Don't link own bins
        .map((dir) => normalizePath(dir))
        .map((target: string) => getPackageBins(target, pkgBinOpts)),
    ))
```

Reply:
I see. Then you did the right thing!

Reply:
other tests are breaking...looking...